mtags/src/main/scala-3/scala/meta/internal/pc/Scala3CompilerWrapper.scala

```diff
@@ -12,8 +12,8 @@ class Scala3CompilerWrapper(driver: InteractiveDriver)
   override def compiler(): InteractiveDriver = driver
 
   override def resetReporter(): Unit = {
-    given ctx as Context = driver.currentCtx
-    ctx.reporter.removeBufferedMessages
+    val ctx = driver.currentCtx
+    ctx.reporter.removeBufferedMessages(using ctx)
```

Comment:
Out of sheer curiosity, was this change needed or is it just a style preference?

Reply:
It's so that we can cross compile 3.0.0-M2 and 3.0.0-M3 - given syntax changed between those two versions.

Reply:
Aha, smart!